Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -185,12 +185,11 @@ export default class MedicationTimelineComponent extends Vue {
   private hasErrors: boolean = false;
 
   private medicationLoaded: boolean = false;
-  private pharmacyLoaded: boolean = false;
```

Comment:
this is still necessary since we dont have the ODR functionality complete

Reply:
this should be handled by a variable in the pharmacy model now!